static/app/views/organizationDetails/body.tsx

```diff
@@ -99,34 +104,35 @@ function DeletionPending({organization}: Props) {
   );
 }
 
-function OrganizationDetailsBody({children, organization}: Props) {
+function OrganizationDetailsBody({children, organization}: BodyProps) {
   const status = organization?.status?.id;
   const routeContext = useRouteContext();
 
-  if (status === 'pending_deletion') {
+  if (organization && status === 'pending_deletion') {
```

Comment:
feels like we should change organization to be optional if we have to check the existence

Reply:
nevermind looking at the wrong props